apps/session-manager/src/__mocks__/repositories/queue-client.mocks.ts

```diff
@@ -0,0 +1,8 @@
+import { QueueClient } from "@azure/storage-queue";
+import { vi } from "vitest";
+
+export const mockSendMessage = vi.fn();
```

Comment:
It should be better to define Parameter and Response types, like [here](https://github.com/pagopa/io-auth-n-identity-domain/blob/465db3717f1a794ebdc09664bdc84d7707f5f489/apps/session-manager/src/__mocks__/services/lollipopService.mocks.ts#L12-L13)

Reply:
Great suggestion, fixed in e449053